.github/copilot-instructions.md

```diff
@@ -0,0 +1,28 @@
+# Fivetran Connector SDK
+A public collection of Python examples, templates, and guides for building custom connectors using the Fivetran Connector SDK. It includes ready-to-run example connectors, best-practice patterns, and quickstart examples to help users understand the usage of Fivetran Connector SDK.
+
+**Primary directories**
+- `template_example_connector/` – canonical template: [connector.py](https://github.com/fivetran/fivetran_connector_sdk/blob/main/template_example_connector/connector.py), [configuration.json](https://github.com/fivetran/fivetran_connector_sdk/blob/main/template_example_connector/configuration.json), `requirements.txt`, [README_template.md](https://github.com/fivetran/fivetran_connector_sdk/blob/main/template_example_connector/README_template.md).
+- `examples/` – quickstarts and pattern examples.
+- `connectors/` – API and specific source examples.
+- `fivetran_platform_features/schema_change/` – shows handling of schema/type changes.
+
+**Runtime & tooling (assume unless PR updates it)**
+- Python supported across 3.9–3.13 
+- Linting: `flake8`.
+- Python standard coding and formatting guidelines are followed
+
+# How to validate PRs
+When a PR changes or adds a connector/example/template, perform these checks:
+- Each connector folder should include:
+  - `connector.py` (imports from `fivetran_connector_sdk`),
+  - `requirements.txt` (Required only when external dependencies are needed),
+  - `configuration.json` (no secrets committed),
+  - README (following the template present at `template_example_connector/README_template.md`).
```

Comment:
@fivetran-sahilkhirwal  did you test it? In my epxperience, referencing other files in the repo does not work.